x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/utils/search_after_bulk_create_factory.ts

```diff
@@ -23,11 +23,33 @@ import type {
   SearchAfterAndBulkCreateParams,
   SearchAfterAndBulkCreateReturnType,
   SignalSourceHit,
+  LoggedRequestsEnabled,
 } from '../types';
 import { withSecuritySpan } from '../../../../utils/with_security_span';
 import type { GenericBulkCreateResponse } from '../factories';
+import type { RulePreviewLoggedRequest } from '../../../../../common/api/detection_engine/rule_preview/rule_preview.gen';
 
 import type { BaseFieldsLatest } from '../../../../../common/api/detection_engine/model/alerts';
+import * as i18n from '../translations';
+
+const createLoggedRequestsConfig = (
+  isLoggedRequestsEnabled: boolean | undefined,
+  sortIds: estypes.SortResults | undefined,
+  page: number
+): LoggedRequestsEnabled | undefined => {
```

Comment:
I think we should change the name of the type `LoggedRequestsEnabled` since it sounds like a boolean but it's a config object

Reply:
renamed in https://github.com/elastic/kibana/pull/203320/commits/dfeb094033b300950caf60e5b97432d735a8ed35